compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -1896,6 +1896,8 @@ class Inliner(call: tpd.Tree, rhsToInline: tpd.Tree)(using Context) {
             level -= 1
             try apply(syms, body)
             finally level += 1
+          case TypeApply(fun, _) =>
```

Comment:
Should we skip over `TypeTree` in general?  Or maybe even `TypTree`s?  I don't mind landing the patch as it is, but I was wondering.

Reply:
I think that is a good idea. I will change it. 

Something like `${ f('x): List[X] }` might fail if `X` is uncompled dependency.